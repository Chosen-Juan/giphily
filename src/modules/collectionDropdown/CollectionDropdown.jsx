import React, { Component } from 'react';
import './styles.scss';

export class CollectionDropdown extends Component {
  render() {
    return (
      <div className='collection-dropdown'>
        <span className='collection-title'>Collections</span>
      </div>
    );
  }
};

export default CollectionDropdown;
