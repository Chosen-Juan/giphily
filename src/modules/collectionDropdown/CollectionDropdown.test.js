import React from 'react';
import { shallow } from 'enzyme';

import { CollectionDropdown } from './CollectionDropdown';

describe('CollectionDropdown', () => {
  it('should match snapshot', () => {
    const collectionDropdown = shallow(<CollectionDropdown />);
    expect(collectionDropdown).toMatchSnapshot();
  });
});
