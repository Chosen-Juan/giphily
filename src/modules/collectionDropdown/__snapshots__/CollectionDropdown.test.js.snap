// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CollectionDropdown should match snapshot 1`] = `
<div
  className="collection-dropdown"
>
  <span
    className="collection-title"
  >
    Collections
  </span>
</div>
`;
